test/e2e/harness.go

```diff
@@ -259,6 +285,8 @@ func GetConfig() E2EConfig {
 	stressTestNumberOfTransactions := int64(10000)
 	stressTestFailureRate := int64(2)
 	stressTestTargetTPS := float64(700)
+	stressTestAsync := os.Getenv("STRESS_TEST_ASYNC") == "true"
```

Comment:
It's part of an E2E config, we could decouple them, but the stress test uses E2E infrastructure because it's part of the E2E suite. Personally I don't mind if we have to move it elsewhere but someone has to make this decision.